content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -20,39 +20,43 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
 
    </admon>
 
-3. Enter the path of the model file as follows:
+3. Enter the path to `dvc.yaml` the model will be added to. Adding your model to
+   non-root `dvc.yaml` can be helpful if you develop this ML model in a specific
+   subfolder or if this repo is a
+   [monorepo](/doc/studio/user-guide/projects-and-experiments/configure-a-project#monorepo).
+
+4. Enter the path of the model file as follows:
 
    - If the model file is in the Git repository (including if it is saved with
      DVC and/or [MLEM]), enter the relative path of the model (from the
      repository root).
-   - Otherwise, enter the URL to the model file in the cloud.
-
-   If the path you entered is a cloud path, Iterative Studio will ask you for
-   the repository path where the dvc reference to the model should be saved.
+   - Otherwise, enter the URL to the model file in the cloud. Iterative Studio
+     will ask you for the repository path where the dvc reference to the model
+     should be saved.
 
-4. Provide labels for your model. For example, if your model is about reviewing
+5. Provide labels for your model. For example, if your model is about reviewing
    sentiment analysis using natural language processing, one of the labels may
    be `nlp`. You can provide multiple labels as a comma separated list. Eg,
    `nlp, sentiment_analysis`.
 
-5. Optionally, add a brief description for your model.
+6. Optionally, add a brief description for your model.
 
-6. Enter a Git commit message. Then, select the branch to commit to. You can
+7. Enter a Git commit message. Then, select the branch to commit to. You can
    commit to either the base branch or a new branch. Iterative Studio will
    commit the changes to the selected branch. If you commit to a new branch,
    Iterative Studio will also create a Git pull request from the new branch to
    the base branch.
 
-7. Now, click on `Commit changes`.
+8. Now, click on `Commit changes`.
 
-8. At this point, the new model appears in the models dashboard.
+9. At this point, the new model appears in the models dashboard.
 
-9. In your Git repository, you will find that an entry for the new model has
-   been created in the `dvc.yaml` file in the repository's root. If you had
-   committed to a new branch, a new pull request (or merge request in the case
-   of GitLab) will also have been created to merge the new branch into the base
-   branch.
-10. If you had added a model from a cloud storage, the following will also
+10. In your Git repository, you will find that an entry for the new model has
+    been created in the `dvc.yaml` file in the repository's root. If you had
+    committed to a new branch, a new pull request (or merge request in the case
+    of GitLab) will also have been created to merge the new branch into the base
+    branch.
+11. If you had added a model from a cloud storage, the following will also
     happen before the commit is created:
```

Comment:
Minor: I don't think these are actual steps, and it makes this list longer than it needs to be. Can we take out the numbering and leave these in as an explanation of what you see after you complete these steps?

Reply:
Thanks for changing this @aguschin! Again, this is minor and not a blocker, but you could do the same for the register and assign pages (they include steps that are really about explaining what you see afterwards).